code/modules/mob/living/living.dm

```diff
@@ -536,6 +536,10 @@
 	if(buckled && last_special <= world.time)
 		resist_buckle()
 
+	//unmuzzling yourself
+	if(is_muzzled())
```

Comment:
Eh, I did a whoopsie there, I'll need to change it anyway. It should be possible to resist out of the muzzle at any point, regardless of whether you're in cuffs or buckled, which won't be possible if I put it under `resist_restraints`, because that only gets called once you're unbuckled.